Corona-Warn-App/src/main/res/values-de/recycler_bin_strings.xml

```diff
@@ -34,7 +34,7 @@
     <!-- XTXT: Recycle Bin - restore -->
     <string name="recycle_bin_restore">"Wiederherstellen"</string>
     <!-- XTXT: Recycle Bin - remove selected item permanently -->
-    <string name="recycle_bin_remove_permanently">"Endgültig löschen"</string>
+    <string name="recycle_bin_remove_permanently">"Alle endgültig löschen"</string>
```

Comment:
This is part of the context menu for the selected item in recycle bin. Word "Alle" or "All" doesn't make sense here.

Reply:
@CV113 @marcauberer as Juraj mentioned this is for deleting an item from the list , does it make sense to say delete ALL ?

Reply:
@KathrinLuecke FYI. What do you think?

Reply:
No, it does not make sense. But I corrected it with https://github.com/corona-warn-app/cwa-app-android/pull/4324. Why does it come up again?

Reply:
The screenshot in the ticket is from the English translation. And indeed English translation is wrong, but this should be fixed by translation delivery for 2.14

Reply:
@KathrinLuecke see the Jira ticket